packages/block-library/src/post-author/block.json

```diff
@@ -34,6 +34,13 @@
 		}
 	},
 	"usesContext": [ "postType", "postId", "queryId" ],
+	"example": {
+		"viewportWidth": 350,
+		"attributes": {
+			"showBio": true,
+			"byline": "Posted by"
```

Comment:
This sentence probably won't be localized, so I think it would be better to register the example via JavaScript, [like the  Paragraph block](https://github.com/WordPress/gutenberg/blob/872f8b37ddb51b3b7f2d16da1138f97d0887df3d/packages/block-library/src/paragraph/index.js#L23-L29).

Reply:
good call!